jvb/src/main/resources/reference.conf

```diff
@@ -312,6 +312,8 @@ videobridge {
   # The experimental multiple streams per endpoint support
   multi-stream {
     enabled = false
+    # Allows mixing source name aware clients together with legacy endpoint ID based in the same conference
+    compatibility-mode = false
```

Comment:
It doesn't look like this is used? (And I don't see any reason why we'd want it to be false, so I think it shouldn't be.)

Reply:
oh I thought I have removed it, thanks